bricks.js

```diff
@@ -208,4 +208,75 @@ class SideTube {
             ctx.strokeRect(this.BB.x - this.game.camera.x, this.BB.y, this.BB.width, this.BB.height);
         }
     };
-};
\ No newline at end of file
+};
+
+class FireBar {
+    constructor(game, x, y) {
+        Object.assign(this, { game, x, y });
+        let brick = new Brick(this.game, this.x, this.y, 3, "None");
+        this.game.addEntity(brick);
+
+        this.angle = 0;
+        this.fires = [];
+        for (var i = 0; i < 6; i++) {
+            var x = i * 25 * Math.sin(this.angle);
+            var y = i * 25 * Math.cos(this.angle);
+            if(i <= 1) {
+                var fire = new Fire(this.game, this.x + x + 12, this.y + y + 12, true);
+            } else {
+                var fire = new Fire(this.game, this.x + x + 12, this.y + y + 12, false);
+            }
+            this.fires.push(fire);
+            this.game.addEntity(fire);
+        }
+    };
+
+    update() {
+        this.angle += this.game.clockTick;
+        if(this.angle >= 360){
+            this.angle = 0;
+        }
+        for (var i = 0; i < 6; i++) {
+            var x = i * 25 * Math.sin(this.angle);
+            var y = i * 25 * Math.cos(this.angle);
+            this.fires[i].x = this.x + x + 12;
+            this.fires[i].y = this.y + y + 12;
+        }
+    };
+
+    drawMinimap(ctx, mmX, mmY) {
+    };
+
+    draw(ctx) {
+        // blockd
+    };
+}
+
+class Fire {
```

Comment:
do note the existence of the "fireball" class added by avatarabo for their "firemario" task (a flower that lets mario shoot fireballs). no conflict here fortunately, but I wanted to mention this when I noticed